src/workers/FetchTiffWorker.ts

```diff
@@ -90,9 +94,11 @@ self.onmessage = async function (e) {
     // deposit in full channel array in the right place
     const offset = zslice * tilesizex * tilesizey;
     if (arrayresult.BYTES_PER_ELEMENT > 4) {
-      console.log("byte size not supported yet");
+      throw new VolumeLoadError("byte size not supported yet", { type: VolumeLoadErrorType.INVALID_METADATA });
```

Comment:
Include the size that was provided in this message for debugging?

Reply:
Sure!